hibernate-infinispan/src/main/java/org/hibernate/cache/infinispan/impl/BaseRegion.java

```diff
@@ -59,12 +59,13 @@
     *
     * @param cache instance for the region
     * @param name of the region
+	 * @param transactionManager transaction manager may be needed even for non-transactional caches.
     * @param factory for this region
     */
-	public BaseRegion(AdvancedCache cache, String name, RegionFactory factory) {
+	public BaseRegion(AdvancedCache cache, String name, TransactionManager transactionManager, RegionFactory factory) {
 		this.cache = cache;
 		this.name = name;
-		this.tm = cache.getTransactionManager();
+		this.tm = transactionManager;
```

Comment:
This is a good improvement, since it means the user only needs to worry about getting the Hibernate transaction manager configuration right for the standalone (outside EE container) use case. :+1: